### ISSUE TITLE
Can't connect

### ISSUE THREAD

**Issue by nathan1050** [MathiasHaudgaard/reflex_behaviour#4]
Hi Mathias,

My Terminal keeps getting stuck at this point and nothing moves further:
'''
(sd_car) n@n-MacBook-Pro reflex_behaviour-master % python drive.py models/model_50000
NOT RECORDING THIS RUN ...
(66957) wsgi starting up on http://0.0.0.0:4567
'''
What could be the issue?
I have the correct version of socket and have installed everything in your environment file.
I have the simulator set to autonomous mode

Many thanks for your help

![Screenshot 2021-08-02 at 8 55 50 PM](https://user-images.githubusercontent.com/62048196/127941155-e6320efb-8b94-4585-8614-f090738abbc7.jpg)

**Comment by MathiasHaudgaard** [comment]
Hi Nathan.

I could replicate the problem. However, I'm not really sure what causes the issue.
I removed the environment and reinstalled it. After that, it worked again. Can you try that out?

**Comment by nathan1050** [comment]
Hi Mathias,

Thank you for your reply.

It hasn't worked out for me still. 

When using the mac, is there any specific location that the simulator.app has to be placed before opening it? Are there any other steps apart from just opening it and running on autonomous mode?

**Comment by MathiasHaudgaard** [comment]
> When using the mac, is there any specific location that the simulator.app has to be placed before opening it?

I don't think so, but honestly idk. I'm using a windows pc and unfortunately, I don't own a mac.

> Are there any other steps apart from just opening it and running on autonomous mode?

No, that basically what's needed. You could try to change the python-socketio version and see if that helps. Or maybe the mac is blocking the connection. 

I'll look more into :shipit: 👍
